src/bindings/js/node/tests/model.test.js

```diff
@@ -111,4 +111,37 @@ describe('Model.getOutputSize()', () => {
   it('should return 1 for the default model', () => {
     assert.strictEqual(model.getOutputSize(), 1, 'Expected getOutputSize to return 1 for the default model');
   });
+});
+
+describe('Model.getOutputElementType()', () => {
+  it('should return a string indicating the element type of the specified output', () => {
+    const result = model.getOutputElementType(0);
+    assert.strictEqual(typeof result, 'string', 'getOutputElementType() should return a string');
+  });
+
+  it('should accept a single number argument', () => {
+    assert.throws(() => {
+      model.getOutputElementType();
+    }, /^Error: Invalid argument. Expected a single number for output index\.$/, 'Expected getOutputElementType to throw an error when called without arguments');
+
+    assert.throws(() => {
+      model.getOutputElementType('unexpected argument');
+    }, /^Error: Invalid argument. Expected a single number for output index\.$/, 'Expected getOutputElementType to throw an error when called with a non-number argument');
+
+    assert.throws(() => {
+      model.getOutputElementType(0, 1);
+    }, /^Error: Invalid argument. Expected a single number for output index\.$/, 'Expected getOutputElementType to throw an error when called with more than one argument');
```

Comment:
This line looks too long. Maximum length is 80 characters.

Reply:
I have shortened the string in the update